components/finance/SummaryInfo/SummaryInfo.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+import SummaryInfo from '.'
+
+export default {
+  title: 'SummaryInfo',
+  component: SummaryInfo,
+}
+
+const Template = () => <SummaryInfo />
```

Comment:
show the summary panel with mock data in storybooks. Fetch the mock data from MSW